Disable the MD5 build sum that breaks all out-of-tree plugins.
It's not a change it should be done on a stable release, anyway...

 -- Faidon Liambotis <paravoid@debian.org>

--- asterisk-1.4.15~dfsg.orig/include/asterisk/module.h
+++ asterisk-1.4.15~dfsg/include/asterisk/module.h
@@ -204,9 +204,6 @@ struct ast_module_info {
 
 	const char *key;
 	unsigned int flags;
-
-	/*! The value of AST_BUILDOPT_SUM when this module was compiled */
-	const char buildopt_sum[33];
 };
 
 void ast_module_register(const struct ast_module_info *);
@@ -234,7 +231,6 @@ void ast_module_unref(struct ast_module 
 		desc,					\
 		keystr,					\
 		flags_to_set,				\
-		AST_BUILDOPT_SUM,			\
 	};						\
 	static void  __attribute__ ((constructor)) __reg_module(void) \
 	{ \
@@ -264,7 +260,6 @@ const static __attribute__((unused)) str
 		.flags = flags_to_set,				\
 		.description = desc,				\
 		.key = keystr,					\
-		.buildopt_sum = AST_BUILDOPT_SUM,		\
 		fields						\
 	};							\
 	static void  __attribute__ ((constructor)) __reg_module(void) \
--- asterisk-1.4.15~dfsg.orig/main/loader.c
+++ asterisk-1.4.15~dfsg/main/loader.c
@@ -76,8 +76,6 @@ static unsigned char expected_key[] =
 { 0x87, 0x76, 0x79, 0x35, 0x23, 0xea, 0x3a, 0xd3,
   0x25, 0x2a, 0xbb, 0x35, 0x87, 0xe4, 0x22, 0x24 };
 
-static char buildopt_sum[33] = AST_BUILDOPT_SUM;
-
 static unsigned int embedding = 1; /* we always start out by registering embedded modules,
 				      since they are here before we dlopen() any
 				   */
@@ -615,13 +613,6 @@ static unsigned int inspect_module(const
 		return 1;
 	}
 
-	if (!ast_strlen_zero(mod->info->buildopt_sum) &&
-	    strcmp(buildopt_sum, mod->info->buildopt_sum)) {
-		ast_log(LOG_WARNING, "Module '%s' was not compiled with the same compile-time options as this version of Asterisk.\n", mod->resource);
-		ast_log(LOG_WARNING, "Module '%s' will not be initialized as it may cause instability.\n", mod->resource);
-		return 1;
-	}
-
 	return 0;
 }
 
